### PR TITLE
3 121 look up table for public transport search interfaces

### PR DESCRIPTION
**Pull request by seitzm97**
Issue resolved. See comments in the script, most APIs are not available and usefull for our purpose.

### CODE REVIEW

File: lookuptable.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May  4 08:50:18 2022
+
+@author: Michael
+"""
+
+class Lookuptable:
+    def __init__(self):
+        self.lookuptable = {
```
Comment (cgurtner) [inline lookuptable.py:10]: This variable could be changed into a constant. It doesn't need to be appended to the instance `self.`.  Furthermore the code indentations isn't uniform.

File: lookuptable.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May  4 08:50:18 2022
+
+@author: Michael
+"""
+
+class Lookuptable:
```
Comment (cgurtner) [inline lookuptable.py:8]: Is Lookuptable an ideal name? I would suggest a name that implicitly says what it contains. E.g. `PublicTransportInterfaces` or `PubliTransportURL` or something like that.

File: lookuptable.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "api": "" #Nur mit Anmeldung
+    },
+    "uk": {
+        "ui": "https://www.nationalrail.co.uk/",
+        "api": ""}
+}
+        
+#test        
+if __name__ == '__main__':
+    lookuptable = Lookuptable()
```
Comment (cgurtner) [inline lookuptable.py:50]: Please add a method to retrieve the data for a specific country. In the end we should be able to import this class, create the instance of it and do something like `urls = lookuptable.get_data('ik'); urls['ui']`.

File: lookuptable.py (hunk 1 above)
Comment (cgurtner) [inline lookuptable.py:10]: This variable could be changed into a constant. It doesn't need to be appended to the instance self.. Furthermore the code indentations isn't uniform.

File: lookuptable.py (hunk 2 above)
Comment (cgurtner) [inline lookuptable.py:8]: Is Lookuptable an ideal name? I would suggest a name that implicitly says what it contains. E.g. `PublicTransportInterfaces` or `PubliTransportURL` or something like that.

**Review by cgurtner — CHANGES_REQUESTED** [review]
review done

File: lookuptable.py (hunk 3 above)
Comment (seitzm97) [reply lookuptable.py:50]: 2 methods added, one to retrieve the URL and one to open said URL. See testing in script.

File: lookuptable.py (hunk 1 above)
Comment (seitzm97) [reply lookuptable.py:10]: Table has been changed in to a constant.

File: lookuptable.py (hunk 2 above)
Comment (seitzm97) [reply lookuptable.py:8]: Name has been changed

**Review by cgurtner — COMMENTED** [review]
submit

**Pull request merged**